api/routes/onboarding.py

```diff
@@ -10,20 +16,85 @@ def onboarding():
     """
     docstring
     """
-    return "Onboarding API. This API endpoint is used for onboarding users and stokvels ( on creation )"
-
+    return render_template("onboarding_template.html")
 
 @onboarding_bp.route(f"{BASE_ROUTE}/users", methods=["POST"])
```

Comment:
Will need a more specific endpoint, users may be abit to general. Something like users/onboarding